GameEngine/GameState.cs

```diff
@@ -24,27 +24,27 @@ public GameState(IPlayer player, Deck deck, int gameSizeMultiplier)
         public void StartGame()
         {
             var hand = Deck.Draw(3);
-            Player.AddCardsToHand(hand);
+            Player.Hand.Add(hand);
```

Comment:
👍 I didn't realize it when you pitched the idea, but the need for a `Hands` class is directly implied by the number of `Player` functions whose name contains the word "Hand". Great job noticing!